woocommerce/woocommerce-android
# [Dynamic Dashboard] Refresh blaze on pull-to-refresh

Part of: #11250 


### Description
This PR adds logic to refresh the Blaze view on pull-to-refresh action.

### Testing instructions
1. Make sure the Dynamic Dashboard feature flag is enabled.
2. Open the app, and sign in using a Jetpack site.
3. Pull the MyStore screen to refresh it.
4. Confirm the Blaze view is refreshed while showing a shimmer effect.

### Images/gif
[blaze-refresh.webm](https://github.com/woocommerce/woocommerce-android/assets/1657201/9c0d96a4-42dc-4a14-9e58-0fe243ec7036)
